src/js/common/components/AlertBox.jsx

```diff
@@ -33,7 +33,7 @@ class AlertBox extends React.Component {
 
   render() {
     if (!this.props.isVisible) {
-      return <div/>;
+      return <div aria-live="assertive"/>;
```

Comment:
yep I added this intentionally so that screen readers can expect that the content inside the div is expected to update/change. When our alert appears, the outer div should be in the same place so I expect most screen readers to recognize it as the same div